.stylelintrc.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+/* eslint-disable-next-line node/no-unpublished-require */
+const { browsers } = require('./tests/dummy/config/targets');
+
+module.exports = {
+  extends: ['stylelint-config-standard'],
+  plugins: ['stylelint-no-unsupported-browser-features', 'stylelint-order'],
+  rules: {
+    /*
+      Customize plugins
+    */
+    'order/properties-order': [
+      [
+        // Defined by ember-css-modules
+        'composes',
+      ],
+      {
+        unspecified: 'bottomAlphabetical',
+      },
+    ],
```

Comment:
This ensures that, in any block, CSS properties are listed in alphabetical order, with `composes` at the top.

https://github.com/hudochenkov/stylelint-order/issues/14#issuecomment-370804987